integration-tests/project-folder/sources/meta-fixtures/rename.bb

```diff
@@ -0,0 +1,13 @@
+foo='foo'
+
+python() {
+    foo='foo'
+    print(foo)
+    d.getVar('foo') # should be included in global variables, but it does not work in integration tests for unknown reasons
```

Comment:
No, this is intended so far, you can remove the comment. Here it is a python string, we would need additional logic to state that some specific functions (getVar, setVar, ...) can have hidden variable symbols as strings.

Reply:
What is not intended?

Currently, renaming `foo` in d.getVar will also rename the `foo` on first line and `foo` in `echo "${foo}"` (it is a bit buggy, but it is fixed in #215). It just does not work in integration tests, and I've not managed to understand why. This line is intended to document this behavior.

Reply:
What I mean is that its OK If `d.getVar('foo')` is not detected as global variable since its a string here. It does references a global variable but our parser needs to know this is a special function which accepts strings that point to variables. I'd remove this comment. If this is not working in integration tests, it's not a limitation IMO. It's a complexity we can have later on, but it's not a bug.

Reply:
Thing is our parser *does* know "this is a special function which accepts strings that point to variables", and the variable does get updated on rename (it even gets updated twice in this PR, but it is fixed in #215).

If some modification ever has for side effect to make it work on integration tests, that line will fail because the test expects an exact number of text edits. The line will remind the person who's doing the PR to add an additional test case to prevent eventual regression, and the comment will explain that the test case did not have the expected behavior before so the person doing the PR does not get confused.